.circleci/base.yml

```diff
@@ -10,6 +10,19 @@ commands:
             git submodule update --init --recursive
 
 jobs:
+  check-format:
+    docker:
+      - image: skiplabs/skdb-base:latest
+    steps:
+      - checkout
+      - run:
+          name: Check code is formatted
+          command: |
+            apt-get install -q -y clang-format-15
+            update-alternatives --install /usr/bin/clang-format clang-format /usr/bin/clang-format-15 100
+            npm install -g prettier
```

Comment:
Note these lines are `[DO NOT COMMIT] temporary per-job setup of auto-formatters` and won't be landed.